README.md

```diff
@@ -610,7 +610,7 @@ Common description. For a detailed description, use `tt help command` .
 -   `check` - check an application file for syntax errors.
 -   `connect` - connect to the tarantool instance.
 -   `rocks` - LuaRocks package manager.
--   `cat` - print into stdout the contents of .snap/.xlog files.
+-   `cat` - print into stderr the contents of .snap/.xlog files.
```

Comment:
`tt cat` should still print snap/xlog file content to stdout, not stderr.

Reply:
fixed